crates/hotshot/src/lib.rs

```diff
@@ -690,20 +690,8 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> ConsensusApi<TYPES, I>
         self.hotshot.config.num_nodes_with_stake
     }
 
-    fn propose_min_round_time(&self) -> Duration {
```

Comment:
H-uh. I guess I should've checked whether someone else was doing this.

Reply:
Oop, me too 🙊